content/intro-to-storybook/svelte/es/get-started.md

```diff
@@ -0,0 +1,166 @@
+---
+title: 'Tutorial Storybook para Svelte'
+tocTitle: 'Empezando'
+description: 'Configurar Svelte Storybook en tu entorno de desarrollo'
+---
+
+Storybook se ejecuta junto con tu aplicación en modo desarrollo. Te ayuda a crear componentes de interfaz de usuario aislados de la lógica y el contexto de tu aplicación. Esta edición de Aprende Storybook es para Vue; existe otras ediciones para [Vue](/vue/es/get-started), [Angular](/angular/es/get-started), [React](/angular/es/get-started) y [React Native](/react-native/es/get-started).
```

Comment:
@IvanSotelo if you can change "Esta edición de Aprende Storybook es para Vue" to "Esta edición de Aprende Storybook es para Svelte" and adjust the links accordingly it would be awesome.